docs/man/qdstat.8.adoc

```diff
@@ -41,6 +41,89 @@ include::{generated_man}/qdstat_help.adoc[]
 OUTPUT COLUMNS
 --------------
 
+qdstat -g
+~~~~~~~~~
+
+Version::
+The version of Dispatch Router.
+
+Mode::
```

Comment:
The router also has a standalone mode 

https://github.com/apache/qpid-dispatch/blob/master/python/qpid_dispatch/management/qdrouter.json#L388